### PR TITLE
Adds a syndicate asteroid base ruin.

### PR DESCRIPTION
**Pull request by WJohn**
Adds a syndicate asteroid base ruin!

This base comes fully functional with life support, power, food, communications, an armory, appropriate accesses, super deadly turrets, lockdown features, and a self destruct button!

The operatives on it, should any ghost choose to spawn as one, are required to stay on station unless they have a very good reason to leave it (such as being ordered to be admins/syndicate command). They could provide intel to traitors/nuke ops should this ruin be picked in that round type.

:cl: WJohn
add: Garbled radio transmissions have been detected in a nearby asteroid field. Centcomm advises caution when exploring near-station space.
/:cl:

### CODE REVIEW

File: code/datums/ruins/space.dm [hunk 1]
```diff
@@ -303,4 +303,12 @@
...
 	description = "Absolutely nothing strange going on here please \
-		move along, plenty more space to see right this way!"
\ No newline at end of file
+		move along, plenty more space to see right this way!"
+
+/datum/map_template/ruin/space/syndicatestation1
+	id = "syndicatestation1"
+	suffix = "syndicatestation1.dmm"
+	name = "Syndicate Asteroid Station"
+	description = "A small, undetected syndicate base built into an asteroid. \
```
Comment (ChangelingRain) [inline code/datums/ruins/space.dm]: This line split is okay, because it's the end of the line.

File: code/datums/ruins/space.dm [hunk 2]
```diff
@@ -303,4 +303,12 @@
...
+		It appears to act as a supply and communications relay for agents \
+		in the field."
```
Comment (ChangelingRain) [inline code/datums/ruins/space.dm]: This one looks ridiculous.

**Comment by Armhulen** [comment]
can this replace the tiny, two room one?

**Comment by WJohn** [comment]
Up to MSO to disable that. I can make a PR to remove the old listening post later.

**Comment by Supermichael777** [comment]
where does that dope pod go?

**Comment by Armhulen** [comment]
also, PLEASE make sure to give proper access. The hotel didn't have any access at merge, the lavabase didn't, and et cet.

**Comment by ma44** [comment]
@Supermichael777 The pod can travel around the stations, like the nook ops ship

**Comment by KorPhaeron** [comment]
This seems kind of redundant

**Comment by WJohn** [comment]
The pod can reach any of the locations the nuke op infiltrator can, as well as where the white ship can dock on the station.

**Comment by KorPhaeron** [comment]
If being a half naked lizard with a spear and sometimes being able to cross literal hell to stab the QM makes people mad, the reaction to spawning as literal nuke ops with a shuttle direct to the station is going to cause blood to boil.

**Comment by WJohn** [comment]
The difference is that ash lizards are allowed to go out and do that. These guys are not unless given express permission.

**Comment by GunHog** [comment]
That does not address the problem of "What do the players do **normally**?" There has to be something to do during _any_ roundtype. The Lavaland base has viro and chem training. This one has nothing to do if there are not other Syndies in the round.

**Comment by WJohn** [comment]
Well there *is* comms equipment, so you can spy on the station. You're right though, it might be a good idea to allow the shuttle to reach other destinations in future PRs. I think it'd be pretty cool if you could dock on other derelicts with the scout there.

**Comment by Armhulen** [comment]
Should be a maximum of two people. Four is, like Kor said, cruel.

**Comment by KorPhaeron** [comment]
Even if it was two guys with a camera it'd be entirely redundant (outside of a promise to later add space exploring) and basically doubling the chance for the syndie camera man to spawn.

**Comment by SgtxRootbeer** [comment]
Actually, this is a good idea. however, it needs modifications...
We have
golems, to teach player the mining/rnd chain
ash walkers, to teach players basic combat and greyshittery
lifebringers, to teach botany
and now, the lavaland syndie base to teach players chem and viro.
this, can be modified to teach players basic engineering by giving them a goal, to make a supermatter/tesla engine combo from scratch. they are only given an rpd or 2, blueprints, a pa crate, and a ton of metal and glass...

**Comment by WJohn** [comment]
There's no guarantee you'll get any agents to spawn in here, let alone all four.

Also, since when was it necessary to make ruins about teaching players how to play the game? You have to be pretty damn experienced to even know how to navigate the menus and *find* this place, or other ruins.

Furthermore you're suggesting a COMPLETELY different ruin, I'm not ripping this thing apart or throwing it all out just to make something entirely different. These things take days to make.

**Comment by Armhulen** [comment]
space bar doesn't get any roles and it's ONE sleeper

**Comment by SgtxRootbeer** [comment]
nonsense, all you have to do to make this an engineering ruin is add some hardsuits, mats, autolathe, circuit boards, pa crate and a giant empty space outside. the idea behind engineering is that you get to build stuff, therefore, its really easy to make an engineering ruin as you dont really have to add anything.

**Comment by WJohn** [comment]
>Nonsense, all you have to do is remake the map.

No. Out.

That's a new ruin entirely, not fit for this PR.

**Comment by SgtxRootbeer** [comment]
as you wish.

**Comment by WJohn** [comment]
I can make another ruin with what you suggested but that's not for here.

**Comment by SgtxRootbeer** [comment]
nah, i wanna try mapping, and i also have a very specific vision for an engineering ruin.

**Comment by kevinz000** [comment]
@KorPhaeron they need to git gud

**Comment by Niknakflak** [comment]
Not a whole lot of point adding this knowing it will only cause problems and headaches and its only purpose is to do this.

**Comment by KingPhilipIII** [comment]
Okay. So I'm always game for more Syndie Station but there really isn't enough to do that's true. Maybe expand the size of the asteroid and add an airlock to the work area. Doesn't even have to be for an engine, just simple construction for the autistic among us. As much as I hate to say it but without a very specific task to fill time it is an awful lot like the lavaland base with less to do. 

Second. Kor's comment about half naked lizards crossing hell to stab the QM made me laugh in the middle of a library. Don't know why. Don't care.

**Comment by WJohn** [comment]
Hm, I think I could expand on this in future PRs to make it more interesting. How about I give the syndicate scout the ability to visit various space ruins, so the syndies have something to do? They'll simply have to avoid the station.

**Comment by RemieRichards** [comment]
We should codeban them from the station Z (for space transitions).

Also I'd be ok with them being able to explore space but not like, the ship going directly to ruins that's terrible. they should get the spacebike vehicles we have (in code, unused besides badminnery (definitely but one on this base regardless of what happens!)) and they should have to explore manually, that's more interesting.

**Comment by optimumtact** [comment]
That's an excellent idea Remie

**Comment by optimumtact** [comment]
https://tgstation13.org/phpBB/viewtopic.php?f=9&t=8783 I got inspired by remie's idea

**Comment by WJohn** [comment]
I think I'll try something different for now.

This toe dipping bullshit with whether or not we can have antagonists who can/can't access the station or interact with players whatsoever is getting on my nerves. I'm gonna ask kor exactly what he wants out of space content and really crack down on this shit and what makes it work from a design/gameplay standpoint if I'm to try mapping or remapping space/lavaland content.

**Comment by FantasticFwoosh** [comment]
TBH Kor has had the most success but probably doesn't know precisely what to give people.

Niches in the audience (i made a point of surgery attachment, xenobio, & genetics etc for my ayy xenobase ruin i put foward on the forums https://tgstation13.org/phpBB/viewtopic.php?f=9&t=8633 etc to cover niches to really explore and elaborate on those areas with new features.) is what id be looking at WJohn.

**Pull request closed without merging**